### PR TITLE
add an mcommunity initializer in /config

### PR DESCRIPTION
**Pull request by gordonleacock**
Based on the ezid.rb initializer

add an mcommunity initializer in /config
update config/mcommunity.yml

### CODE REVIEW

**Comment by gordonleacock** [comment]
Arggg: /home/travis/build/mlibrary/umrdr/vendor/bundle/ruby/2.2.0/gems/activesupport-4.2.6/lib/active_support/dependencies.rb:268:in `load': /home/travis/build/mlibrary/umrdr/config/initializers/mcommunity.rb:40: syntax error, unexpected keyword_end, expecting end-of-input (SyntaxError)

Working on it...

**Comment by gordonleacock** [comment]
Hmmm, /home/travis/build/mlibrary/umrdr/config/initializers/mcommunity.rb:34:in `<top (required)>': uninitialized constant LdapConfig (NameError)

**Comment by grosscol** [comment]
The `mcommunity.yml` should be `mcommunity.yml.sample`.  The bin/setup task will make a mcommunity.yml file if one does not exist.

This PR depends on #296

**Comment by gordonleacock** [comment]
Well that's weird, I have mcommunity.yml in my .gitignore file and I have an mcommunity.yml.sample file that is NOT in .gitignore. Let me see if I can clean this up.

File: config/initializers/mcommunity.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    raise "The #{::Rails.env} environment settings were not found in #{cfg_file}" unless mcommunity_yml[::Rails.env]
+    mcommunity_cfg = mcommunity_yml[::Rails.env].symbolize_keys
+  end
+
+  mcommunity_cfg
+end
+
+
+yml_cfg = load_yml
+LdapConfig = Umrdr::LdapConfig.new
```
Comment (grosscol) [inline config/initializers/mcommunity.rb]: This is not needed.  You're creating an instance of Umrdr::LdapConfig and assigning it to a variable that makes it look like a class.

File: config/initializers/mcommunity.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+LdapConfig.host     = yml_cfg[:host] || 'ldap.umich.edu'
```
Comment (grosscol) [inline config/initializers/mcommunity.rb]: This needs to be using a the class method.  You've assigned an instance of Umrdr::LdapConfig to LdapConfig.  There is no `host=` instance method.  That should be a class method.

Try `Umrdr::LdapConfig.host=`

**Comment by grosscol** [comment]
Merged #296.  Try rebasing and see if the specs pass.

**Pull request merged**